app/lib/features/chat_ng/widgets/message_actions_widget.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:acter/features/chat_ng/actions/copy_message_action.dart';
+import 'package:acter/features/chat_ng/actions/redact_message_action.dart';
+import 'package:acter/features/chat_ng/actions/report_message_action.dart';
+import 'package:acter/features/chat_ng/providers/chat_room_messages_provider.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show RoomEventItem;
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class MessageActionsWidget extends ConsumerWidget {
```

Comment:
This widget is responsible for rendering message actions options .i.e. copy/delete/reply/edit etc.